### PR TITLE
Verifier cleanup/broken verification without workaround

### PR DESCRIPTION
**Pull request by Jakub89**
Resolves:
https://codepoets.atlassian.net/browse/GF-109

### CODE REVIEW

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py [hunk 1]
```diff
@@ -193,3 +195,66 @@ def compare_images(image_a, image_b, metrics) -> Dict:
...
+
+
+def convert_image_to_simple_array(image: np.ndarray) -> np.ndarray:
+    """
+    Function for squashing image array in to 0/1 array, which translate to
+    black/white pixel
+    """
+    simpe_array = np.zeros((image.shape[0], image.shape[1]))
+    for x, row in enumerate(image):
+        for y, pixel_array in enumerate(row):
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: isn't `pixel_array` just a `pixel` or `single_pixel`?

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py [hunk 2]
```diff
@@ -193,3 +195,66 @@ def compare_images(image_a, image_b, metrics) -> Dict:
...
+            pixel_sum = np.sum(pixel_array)
+            if pixel_sum <= 30:
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: please use `pixel_sum <= WHITE + OFFSET`

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py [hunk 3]
```diff
@@ -193,3 +195,66 @@ def compare_images(image_a, image_b, metrics) -> Dict:
...
+                simpe_array[x, y] = 0
+            elif pixel_sum >= 735:
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: please use `pixel_sum >= BLACK - OFFSET`

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py [hunk 4]
```diff
@@ -193,3 +195,66 @@ def compare_images(image_a, image_b, metrics) -> Dict:
...
+                simpe_array[x, y] = 1
+            else:
+                raise Exception("Pixel incomparable. Neither black or white.")
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: maybe `ValueError`?

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py [hunk 5]
```diff
@@ -65,7 +66,7 @@ def prepare_params(mounted_paths, subtask_border, scene_file_path, resolution, s
     return crops, params
 
 
-def make_verdict( subtask_file_paths, crops, results ):
+def make_verdict(subtask_file_paths, crops, results, raw_verification):
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py]: hm... maybe `use_raw_verification`?
```suggestion
def make_verdict(subtask_file_paths, crops, results, use_raw_verification):
```

File: golem/testutils.py [hunk 6]
```diff
@@ -116,8 +116,9 @@ def additional_dir_content(self, file_num_list, dir_=None, results=None,
         return results
 
     def __remove_files(self):
-        if os.path.isdir(self.tempdir):
-            shutil.rmtree(self.tempdir)
+        pass
+        # if os.path.isdir(self.tempdir):
```
Comment (dybi) [inline golem/testutils.py]: I quess this should not be commited -> pls drop `[TMP] ...` commit

File: golem/verificator/blender_verifier.py [hunk 7]
```diff
@@ -19,6 +19,8 @@
 class BlenderVerifier(FrameRenderingVerifier):
     DOCKER_NAME = 'golemfactory/blender_verifier'
     DOCKER_TAG = '1.1'
+    DOCKER_NAME = 'test'
```
Comment (dybi) [inline golem/verificator/blender_verifier.py]: same as above

**Review by dybi — COMMENTED** [review]
Few things to clarify
baba_jaga_mysli

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py (hunk 1 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: True, it's a single pixel. I'll fix it.

File: golem/testutils.py (hunk 6 above)
Comment (Jakub89) [reply golem/testutils.py]: Yep, I forget about that.

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py (hunk 1 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: Done

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py (hunk 2 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: Done

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py (hunk 3 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: Done

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py (hunk 4 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: Done

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py (hunk 5 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py]: Done

File: golem/testutils.py (hunk 6 above)
Comment (Jakub89) [reply golem/testutils.py]: Done. Removed.

File: golem/verificator/blender_verifier.py (hunk 7 above)
Comment (Jakub89) [reply golem/verificator/blender_verifier.py]: Done. Removed.

**Review by dybi — APPROVED** [review]
Make sure that `use_raw_verification` is used and it's good to go from my PoV ;)
red_afro_approves

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py [hunk 8]
```diff
@@ -4,7 +4,8 @@
 from ..render_tools import blender_render as blender
 from .crop_generator import WORK_DIR, OUTPUT_DIR, SubImage, Region, PixelRegion, \
     generate_single_random_crop_data, Crop
-from .img_metrics_calculator import calculate_metrics
+from .img_metrics_calculator import calculate_metrics, get_raw_verification
```
Comment (kbeker) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py:7]: I wanted to say that style but i forgot that we are in different repo :D

**Review by kbeker — APPROVED** [review]
For me ok :)

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py (hunk 8 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/verificator.py:7]: tenor

**Pull request merged**